wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -971,6 +992,32 @@ class SwiftGenerator private constructor(
     )
   }
 
+  // private fun TypeSpec.Builder.generateMessageDefaults(
```

Comment:
will delete before merging, just keeping it here for the review in case we need it.